charts/opentelemetry-collector/README.md

```diff
@@ -190,6 +190,91 @@ presets:
     extractAllPodAnnotations: true
 ```
 
+### Configuration for Annotation-Based Discovery
+
+The collector can be configured to automatically discover and collect telemetry from pods based on annotations. This feature provides a drop-in replacement for the `logsCollection` preset, allowing for selective collection of logs and metrics from specific pods rather than collecting from all pods in the cluster.
+
+> [!WARNING]
+> Annotation-based discovery and `logsCollection` are mutually exclusive. You should use either:
+>
+> - `presets.logsCollection.enabled: true` (collects logs from all pods)
+> - `presets.annotationDiscovery.logs.enabled: true` (collects logs only from pods with specific annotations)
+>
+> Using both simultaneously will result in configuration conflicts.
+
+To enable this feature, set the `presets.annotationDiscovery.logs.enabled` and/or `presets.annotationDiscovery.metrics.enabled` properties to `true`.
+
+Here is an example `values.yaml`:
+
+```yaml
+mode: daemonset
+
+image:
+  repository: "ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-k8s"
+
+command:
+  name: "otelcol-k8s"
+
+presets:
+  annotationDiscovery:
+    logs:
+      enabled: true
+    metrics:
+      enabled: true
+```
+
+#### How Annotation-Based Discovery Works
+
+When annotation-based discovery is enabled, the collector will:
+
+1. **Discover Pods**: Use the Receiver Creator receiver to watch for pods with specific annotations
+2. **Generate Receiver Configurations**: Automatically generate receiver configuration
+
+**Default Behavior**: When `presets.annotationDiscovery.logs.enabled` is `true`, the collector will collect logs from all containers by default, unless a pod explicitly opts out using the `io.opentelemetry.discovery.logs/enabled: "false"` annotation.
+
+This approach provides the same functionality as `logsCollection` but with fine-grained control over which pods are monitored, making it ideal for environments where you want to selectively collect telemetry from specific applications or services.
+
+#### Customizing Receiver Configuration
```

Comment:
I'd would skip this part. We can just reference the respective Collector's docs: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/receivercreator/README.md#generate-receiver-configurations-from-provided-hints

Here we only care about the preset not the actual Collector's feature, right?